### ISSUE TITLE
Operation Make David's Life A Bit Easier

### ISSUE THREAD

**Issue by ikeydoherty** [solus-project/budgie-desktop#465]
TL;DR The Debian thing is really dragging out and feels like slow open heart surgery right now.
Let's get things on track and centralise communications so that @fossfreedom can get the Debian stuff sorted, and I can go back to other things.

* [x] @fossfreedom to provide a complete list of actions wanted by Debian folks
* [x] Ikey to um and ah about it and probably creating a Debian-Make-Happy script for release testing
* [x] Ikey to provide new release intended for inclusion in Debian, with the release being done in the next couple of weeks.

All Debian-centric issues should live here for now. With all due respect to the Debian folks, getting one snippet at a time drawn out over time is painful, let's get a complete action plan here so I can get my development time back. i.e. kick things into gear.

**Disclaimer**: I will not have anything to do with the maintenance of packages within Debian, this is simply to help David out. I have my own OS to maintain.

**Comment by ikeydoherty** [comment]
Also note the outcome of this action plan *must* comply with the [Budgie Desktop integration practices](https://github.com/solus-project/budgie-desktop/blob/master/README.md#reporting-issues--project-integration)

**Comment by fossfreedom** [comment]
well hurrah!  Debian has today accepted the package into the New queue.  It should clear FTP masters fairly soon and then enter Sid and Ubuntu 16.10.

I'll upload 10.2.6 as soon as FTP masters accepts 10.2.5

thanks for your help with this one @ikeydoherty 

This issue can be closed now.

**Comment by ikeydoherty** [comment]
Amazing work - congrats !

**Comment by LocutusOfBorg** [comment]
@fossfreedom you can give m the 10.2.6 dsc file right now, and I'll sponsor that one too.
(please remember, your sponsor is *always* watching you :) )

**Comment by LocutusOfBorg** [comment]
oops not accepted yet, hold on, but feel free to prepare the dsc in the meanwhile, just in case ftpmasters accepts it

**Comment by ikeydoherty** [comment]
@LocutusOfBorg that was well played sir :P I'd certainly recommend migrating to 10.2.6 as soon as humanly possible over 10.2.5 due to the changes iterated here: https://github.com/solus-project/budgie-desktop/releases/tag/v10.2.6

One of immense importance is fixing out of tree applet builds, which will allow the development and distribution of applets/extensions to Budgie

**Comment by LocutusOfBorg** [comment]
sigh @fossfreedom
```
unfortunately I have to reject your package. Please take care of:
 W: budgie-core: non-standard-dir-perm etc/xdg/autostart/ 0775 != 0755
 W: budgie-core: non-standard-dir-perm usr/bin/ 0775 != 0755
 W: budgie-core: non-standard-dir-perm usr/lib/budgie-desktop/ 0775 != 0755
 W: budgie-core: non-standard-dir-perm usr/lib/budgie-desktop/plugins/ 0775 != 0755
 W: budgie-core: non-standard-dir-perm usr/share/applications/ 0775 != 0755
 W: budgie-core: non-standard-dir-perm usr/share/glib-2.0/ 0775 != 0755
 W: budgie-core: non-standard-dir-perm usr/share/glib-2.0/schemas/ 0775 != 0755
 W: budgie-core: non-standard-dir-perm usr/share/gnome-session/ 0775 != 0755
 W: budgie-core: non-standard-dir-perm usr/share/gnome-session/sessions/ 0775 != 0755
 W: budgie-core: non-standard-dir-perm usr/share/icons/ 0775 != 0755
 W: budgie-core: non-standard-dir-perm usr/share/icons/hicolor/ 0775 != 0755
 W: budgie-core: non-standard-dir-perm usr/share/icons/hicolor/scalable/ 0775 != 0755
 W: budgie-core: non-standard-dir-perm usr/share/icons/hicolor/scalable/actions/ 0775 != 0755
```

**Comment by ikeydoherty** [comment]
Is it possible you have an incorrect umask before building the package? Those directories *should* be 00755 like they are on Solus

```
 ikey@solus-bdw  ~  stat -c "%a %n" /usr/lib/budgie-desktop/plugins 
755 /usr/lib/budgie-desktop/plugins
 ikey@solus-bdw  ~  
```

**Comment by LocutusOfBorg** [comment]
http://debomatic-amd64.debian.net/distribution#unstable/budgie-desktop/10.2.5-1/lintian
http://debomatic-amd64.debian.net/distribution#unstable/budgie-desktop/10.2.5-1/piuparts

please fix them too

**Comment by fossfreedom** [comment]
so close ... will upload soon.

**Comment by ikeydoherty** [comment]
^ Chucked in a quick patch for you to help - we never explicitly linked because libbudgietheme is only used from GTK linked components, but it'll make your error go away in Debian

**Comment by LocutusOfBorg** [comment]
In Unstable, probably in Ubuntu yakkety in around two days

**Comment by ikeydoherty** [comment]
Awesome! @LocutusOfBorg what's the policy on minor updates in Ubuntu then? I imagine we might just need a `10.2.7` after all..

**Comment by LocutusOfBorg** [comment]
I need a file from @fossfreedom to sponsor.

**Comment by LocutusOfBorg** [comment]
And some minutes / good connection for checks and upload

**Comment by ikeydoherty** [comment]
K well I leave this in the hands of you guys :]

**Comment by LocutusOfBorg** [comment]
I'm just a sponsor I just sign check and upload what people give me

**Comment by LocutusOfBorg** [comment]
Yeah, it migrated into Ubuntu yakkety a few seconds ago.
